### PR TITLE
Add unique container names for leftover services

### PR DESCRIPTION
**Pull request by BerniVarga**
-> Add unique container names for alphaville-series & mongo-backup
-> Fix neo4j unique container name generation (add "startsWith" pattern in filter)
-> Update service file template

Tested in xp.

⚠️ Note: a failover needs to be done, when deploying to prod

### CODE REVIEW

**Comment by catalin-ilea** [comment]
+1

**Comment by kovacshuni** [comment]
+1

File: alphaville-series-rw-neo4j-blue@.service [hunk 1]
```diff
@@ -9,15 +9,21 @@ Environment="DOCKER_APP_VERSION=latest"
 TimeoutStartSec=0
 # Change killmode from "control-group" to "none" to let Docker remove work correctly.
 KillMode=none
-ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
-ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p-%i_)" > /dev/null 2>&1'
```
Comment (tosan88) [inline alphaville-series-rw-neo4j-blue@.service]: Can we please have the same pattern for blue and red services?

**Comment by tosan88** [comment]
:+1:

**Pull request merged**